build_tools/python/e2e_test_framework/models/matmul.py

```diff
@@ -0,0 +1,69 @@
+## Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Defines Matmul microbenchmarks."""
+
+from e2e_test_framework import unique_ids
+from e2e_test_framework.definitions import common_definitions
+
+MATMUL_3456X1024X2048_FP16_MLIR = common_definitions.Model(
+    id=unique_ids.UB_MATMUL_3456X1024X2048_FP16_MLIR,
```

Comment:
"UB" to me means "Undefined Behavior". I think here it's maybe representing "μB"? But μ!=U. I'd prefer "M" in that case (which is actually what an uppercase "μ" looks like). But maybe just spell it out? Given that everything here is a benchmark, how about just "MICRO"?

```suggestion
    id=unique_ids.MICRO_MATMUL_3456X1024X2048_FP16_MLIR,
```

Reply:
Renamed to micro